packages/intrepid2/src/Projection/Intrepid2_LagrangianInterpolationDef.hpp

```diff
@@ -56,6 +56,217 @@
 namespace Intrepid2 {
 namespace Experimental {
 
+
+
+template<typename scalarViewType,
+typename ortViewType,
+typename t2oViewType,
+typename subcellParamViewType,
+typename intViewType>
+struct F_computeDofCoordsAdCoeff {
+  typedef typename scalarViewType::value_type value_type;
+
+  scalarViewType dofCoords, dofCoeffs;
+  const ortViewType orts;
+  const t2oViewType tagToOrdinal;
+  const subcellParamViewType edgeParam, faceParam;
+  const intViewType edgeInternalDofOrdinals, facesInternalDofOrdinals;
+  const scalarViewType edgeInternalDofCoords, edgeDofCoeffs, ortJacobianEdge, refEdgesTan, refEdgesNormal;
+  const scalarViewType facesInternalDofCoords, faceDofCoeffs, ortJacobianFace, refFaceTangents, refFacesNormal;
+  scalarViewType edgeWorkView, faceWorkView;
+  const ordinal_type cellDim, numEdges, numFaces;
+  const ordinal_type edgeTopoKey, numEdgeInternalDofs;
+  const intViewType faceTopoKey, numFacesInternalDofs;
+  const value_type edgeScale, faceScale;
+  const bool isBasisHCURL, isBasisHDIV;
+
+  F_computeDofCoordsAdCoeff( scalarViewType dofCoords_,
+      scalarViewType dofCoeffs_,
```

Comment:
Isn't the usual convention to use trailing underscores for member variables?  You seem to have reversed this here, leaving member variables unadorned, and using trailing underscores in the constructor argument.

Reply:
Not sure. It's the same in Intrepid2_OrientationToolsDefModifyBasis.hpp. But I can change it if it is better.

Reply:
It is at least a fairly common convention to do what I suggested (though whether underscores lead or trail varies).  I think it is better; it allows the reader to distinguish between member variables and variables that are local to a method at a glance.

Reply:
OK